modules/autotagging-commons/src/main/java/org/opensearch/rule/feature_value_resolver/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+ * This package contains classes to resolve feature value
+ */
+package org.opensearch.rule.feature_value_resolver;
```

Comment:
nit: Google Java Style guide recommends not underscores in package names: https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names